app/ui.js

```diff
@@ -180,7 +180,6 @@ var UI;
             UI.initSetting('view_only', false);
             UI.initSetting('path', 'websockify');
             UI.initSetting('repeaterID', '');
-            UI.initSetting('token', '');
```

Comment:
We currently have deployed noVNC on a bastion host, with a local change that makes host and port hidden inputs, and renames the label of the token. Tokens map 1:1 to available VNC ports of internal hosts (via websockify configuration file that lists them all), so that we can run one websockify for all of them.

While I agree that this setting is confusing to users, and should not be exposed by default, keeping the token as a hidden input (so that we can turn it into a regular input and change the label locally) would help us.